runtime/compiler/optimizer/VectorAPIExpansion.hpp

```diff
@@ -472,6 +472,9 @@ class TR_VectorAPIExpansion : public TR::Optimization
     *  \param objectType
     *     Object type
     *
+    *  \param sourceType
+    *     find source type
```

Comment:
I think this was meant to be added to the doc comment for `isVectorizedOrScalarizedNode()` above (which doesn't list any of the existing parameters)

Reply:
Fixed